labman/gui/templates/library_pooling.html

```diff
@@ -193,21 +221,59 @@
           }
        },
     };
-   
-    var histlayout = {
+    var histogramLayout = {
       autosize: false,
       width: heatwidth,
       xaxis: {title: "Concentration"},
       yaxis: {title: "Number Samples"},
       title: 'DNA Concentration Distribution'
     };
-    var histdata = [trace, blank_trace];
+    var histogramData = [nonBlankData, blankData];
+
+    Plotly.plot($heatmap.attr('id'), heatmapData, heatmapLayout);
+    Plotly.plot($histogram.attr('id'), histogramData, histogramLayout);
+
+    // setup some interactivity to clip the values in the histogram
```

Comment:
Does this want to be simplified into a text box? The advantage is that it wouldn't require any default max values.